tests/algorithms/test_bc.py

```diff
@@ -5,7 +5,8 @@
 
 import pytest
 import torch as th
-from stable_baselines3.common import vec_env
+import utils
```

Comment:
I'm a bit worried about this top-level import -- is this officially supported by `pytest`, can we depend on `sys.pythonpath` being set to include `tests/`?

https://stackoverflow.com/questions/33508060/create-and-import-helper-functions-in-tests-without-creating-packages-in-test-di has some other ideas on how to share helper methods between tests. I think the helper class in `conftest` might be slightly better? But it is also quite hacky, really.

Reply:
It's also a bit odd this is in the "third-party package" section of imports, despite being local... should probably add it to `known_first_party` in `isort` if we do stick with this.

Reply:
I agree putting it in `src/imitation/testing/` would be a good solution, it is a helpful utility.

Reply:
done!